### PR TITLE
Script to make nuget packages for SqlTools ServiceLayer

### PR DESCRIPTION
**Pull request by KevinRansom**
This adds a script and a set of projects that produce a set of nuget packages containing the platform specific. SQL kernel implementations.

run the command:
    buildSqlTools.cmd
and wait.

To test it:
#i "nuget:C:\kevinransom\interactive\artifacts\packages\release\Shipping"
#r "nuget:Microsoft.SqlTools.ServiceLayer,3.0.0-release.49"

Note that your nuget cache will now contain a bunch od ServiceLayer packages.

I will work with Jon tomorrow, to figure out how to modify the sql kernel to make use of these packages.

In order to get package signing cibuild.ps1 has been updated to generate the packages.

### CODE REVIEW

File: src/Microsoft.DotNet.Interactive.ExtensionLab/Microsoft.SqlTools.ServiceLayer/runtime.win-x86.runtime.native.Microsoft.SqlTools.ServiceLayer/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
```
Comment (colombod) [inline src/Microsoft.DotNet.Interactive.ExtensionLab/Microsoft.SqlTools.ServiceLayer/runtime.win-x86.runtime.native.Microsoft.SqlTools.ServiceLayer/Class1.cs:1]: do we need this file?

File: src/Microsoft.DotNet.Interactive.ExtensionLab/Microsoft.SqlTools.ServiceLayer/runtime.win10-arm.runtime.native.Microsoft.SqlTools.ServiceLayer/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
```
Comment (colombod) [inline src/Microsoft.DotNet.Interactive.ExtensionLab/Microsoft.SqlTools.ServiceLayer/runtime.win10-arm.runtime.native.Microsoft.SqlTools.ServiceLayer/Class1.cs:1]: same

File: temp [hunk 3]
```diff
@@ -0,0 +1,6 @@
+runtime.osx-x64.runtime.native.Microsoft.SqlTools.ServiceLayer
```
Comment (colombod) [inline temp:1]: What is this temp file?

**Review by colombod — COMMENTED** [review]
looks fine but I have questions about all class.cs files and presence of a temp file, do we need those one checked in?

File: src/Microsoft.DotNet.Interactive.ExtensionLab/Microsoft.SqlTools.ServiceLayer/runtime.win-x86.runtime.native.Microsoft.SqlTools.ServiceLayer/Class1.cs (hunk 1 above)
Comment (KevinRansom) [reply src/Microsoft.DotNet.Interactive.ExtensionLab/Microsoft.SqlTools.ServiceLayer/runtime.win-x86.runtime.native.Microsoft.SqlTools.ServiceLayer/Class1.cs:1]: The temp file no. I will can it.  The class1.cs files, yes.

File: temp (hunk 3 above)
Comment (KevinRansom) [reply temp:1]: I will remove it

**Review by jonsequitur — APPROVED** [review]

**Comment by KevinRansom** [comment]
Don't merge it, it's got a bug :-)

**Comment by jonsequitur** [comment]
Wouldn't dream of it.

**Comment by KevinRansom** [comment]
Phew !!! :-)

**Pull request merged**